src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -20,6 +20,7 @@
 import static org.hamcrest.core.IsNull.notNullValue;
 import static org.hamcrest.core.IsNull.nullValue;
 import static org.junit.Assert.assertThat;
+import static org.mongojack.TestDBUpdateSerialization.NestedIdFieldWithDifferentType.NESTED_ID_FIELD_VALUE;
```

Comment:
It's needed as in if you take it out you get a compile error, since the `NESTED_ID_FIELD_VALUE` is defined in the inner class `NestedIdFieldWithDifferentType` but used in the test-class. I opted to statically import it since L190 would otherwise get especially long, referencing the field including the name of the inner class.